scripts/rebuilder.py

```diff
@@ -369,6 +380,12 @@ def main():
         pkg_failures = get_monthly_rebuild_regressions(
             project_owner, project_name, start_time, copr_pkgs
         )
+        get_chroot_results(pkg_failures, copr_client)
+        # Delete attributes we don't need to print
+        for p in pkg_failures:
+            for k in ["fail_id", "chroots"]:
+                del p[k]
```

Comment:
Why don't we need to print these?

Reply:
It's just extra information that we don't need, but I guess it wouldn't hurt to include them.